Makefile

```diff
@@ -72,15 +72,15 @@ create_qa:
 		--role arn:aws:iam::946183545209:role/lambda-full-access \
 		--zip-file fileb://./deploy/rhrc.zip \
 		--profile qa
```

Comment:
profile qa ... see comment below. this should match.

Reply:
Good point. See above.